src/TopicSubscription.php

```diff
@@ -104,6 +102,7 @@ public function getQualityOfServiceLevel(): int
      *
      * @return int|null
      */
+    // FIXME: do we need this? the acknowledged one is the one.
```

Comment:
I only added it since we receive this information in CONNACK from the broker. I don't think it is used currently. It could be used in the logic we discussed to remove earlier today. But besides that, I don't see a good use for it either. It doesn't really cost a lot of memory though, so I'd probably just keep it until we find a good use for it. 👍